pymc/model.py

```diff
@@ -1691,11 +1691,11 @@ def point_logps(self, point=None, round_vals=2):
 
         Parameters
         ----------
-        point: Point
+        point: Point, default 'None'
```

Comment:
```suggestion
        point: Point, optional
```

The default isn't really None as explained in the description, None is used as a placeholder to indicate that the attribute `model.initial_point` should be used. In such cases optional instead of default should be used. Reference https://pymc-data-umbrella.xyz/en/latest/sprint/docstring_tutorial.html#parameters